### ISSUE TITLE
Linking to Acclaim has too much badging

### ISSUE THREAD

**Issue by v-stsier** [NewmanCatalysis/ITProCareerCenter#14]
Need Teresa to give direction on what to do with this suggestion:

f.	The linking to Acclaim is filled with badging:  the URLs the site text.  Let’s explore the user flow for a sec…. [tc] I thought the same thing and wanted to see if others felt same way .  We’ll change to your suggestion.  
i.	From ITPro Career Center, select an ‘Explore Cloud Role Salaries’.  I selected IT Security Architect.
ii.	I’m taken to a Badge page for IT Security Architect.  
1.	No salary info here.   this is what the user was promised
2.	Lots of info on Badges, who it’s issued by, what it takes to earn the badge, learn about jobs and employer demand for this badge…..
3.	I can select ‘View details’ under Related Jobs to get Job Openings and Salary Ranges
iii.	I think the link from ITPro Career Center, in ‘Explore Cloud Role Salaries’ needs to go directly to the Acclaim Job Openings and Salary Ranges pages, not the Acclaim Badge pages.
g.	I’ve already Registered for the ITPro Career Center, and I need to Register again for the Microsoft IT Pro Could Essentials program?

SUGGESTED BY: Julie Parsons

**Comment by NewmanCatalysis** [comment]
I agree with the salary feedback.  I actually thought that I'd see cloud role salaries myself.  We'll need acclaim to give us an adjusted role-to-salary-url list in order to update this.

**Comment by Teresaconte** [comment]
We should keep this open as I'll be getting the URLS.  Or maybe I'm reading the GitHub update wrong thx

**Comment by Teresaconte** [comment]
I sent the new links to Newman in email.  Let me know if I need to paste them here.

**Comment by NewmanCatalysis** [comment]
I closed this by mistake and then re-opened.
Cheers,
Newman



Sent from a phone which is more computer than phone.


Thursday, April 7, 2016, 17:28 -0700 from notifications@github.com  <notifications@github.com>:
